docs/README.md

```diff
@@ -1,121 +1,240 @@
 
-### Build
+The benchmarking tools require the following:
+
+- A client to run the benchmarking tools
+- A target database
+  - For a list of databases that ScalarDB supports, see [Supported Databases](https://github.com/scalar-labs/scalardb/blob/master/docs/scalardb-supported-databases.md).
+
+## Set up the benchmarking tools
+
+The following sections describe how to set up the benchmarking tools.
+
+### Clone the ScalarDB benchmarks repository
+
+Open **Terminal**, then clone the ScalarDB benchmarks repository by running the following command:
+
+```console
+$ git clone https://github.com/scalar-labs/scalardb-benchmarks
+```
+
+Then, go to the directory that contains the benchmarking files by running the following command:
 
 ```console
-./gradlew shadowJar
+$ cd scalardb-benchmarks
+```
+
+### Build the tools
+
+To build the benchmarking tools, run the following command:
+
+```console
+$ ./gradlew shadowJar
 ```
 
 ### Create tables
 
-Before loading initial data, the tables must be defined using [ScalarDB Schema Loader](https://github.com/scalar-labs/scalardb/blob/master/docs/schema-loader.md). Get the latest schema loader [here](https://github.com/scalar-labs/scalardb/releases) and execute it with the workload-specific schema file. For setting ScalarDB properties, see also the related documents [here](https://github.com/scalar-labs/scalardb#docs).
+Before loading the initial data, the tables must be defined by using the [ScalarDB Schema Loader](https://github.com/scalar-labs/scalardb/blob/master/docs/schema-loader.md). To apply the schema, go to the [ScalarDB Releases](https://github.com/scalar-labs/scalardb/releases) page and download the ScalarDB Schema Loader that matches the version of ScalarDB that you are using to the `scalardb-benchmarks` root folder.
+
+In addition, you need a properties file that contains database configurations for ScalarDB. For details about configuring the ScalarDB properties file, see [ScalarDB Configurations](https://github.com/scalar-labs/scalardb/blob/master/docs/configurations.md).
+
+After applying the schema and configuring the properties file, select a benchmark and follow the instructions to create the tables.
+
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'TPC-C_1', 'tabset-1')" id="defaultOpen-1">TPC-C</button>
+  <button class="tablinks" onclick="openTab(event, 'YCSB_1', 'tabset-1')">YCSB</button>
+  <button class="tablinks" onclick="openTab(event, 'multi-storage_YCSB_1', 'tabset-1')">Multi-storage YCSB</button>
+</div>
+
+<div id="TPC-C_1" class="tabcontent" markdown="1">
+
+To create tables for TPC-C benchmarking ([`tpcc-schema.json`](https://github.com/scalar-labs/scalardb-benchmarks/blob/master/tpcc-schema.json)), run the following command, replacing the contents in the angle brackets as described:
+
+```console
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config /<PATH_TO>/scalardb.properties -f tpcc-schema.json --coordinator
```

Comment:
[minor] Users can specify an arbitrary properties file name, so probably just `--config <SCALARDB_PROPERTIES_FILE_PATH>` would be also okay.

Reply:
Right. Also, I noticed that if users prepare and use a ScalarDB property file here, it's easy and straightforward to specify the file in the benchmark configuration file. So, specifying the property file as default like the following might be better. Josh, what do you think?

```
[database_config]
config_file = "/<PATH_TO>/scalardb.properties"
#contact_points = "localhost"
#contact_port = 9042
#username = "cassandra"
#password = "cassandra"
#storage = "cassandra"
```

Reply:
@komamitsu Good catch! Since the properties file will depend on what the user has named the file, your suggestion makes sense. I've applied it here and in other parts of this document.

@jnmt I think moving that line to the top would be a good idea. When I set this up in my environment, I wondered if using the properties file was the preferred method since I imagine it would be easier than checking my database configurations under the `[database_config]` section.

For consistency, how about we also move the path to the ScalarDB properties file to the top of the `[database_config]` section in the the three **.toml** files?

Reply:
@josh-wong That's a good idea, and I appreciate it if you could do it.

Reply:
@jnmt No problem! I've updated the three **.toml** files. PTAL!